docker-compose.yml

```diff
@@ -58,7 +58,7 @@ geoserver:
 geowebcache:
   image: georchestra/geowebcache:latest
   volumes:
-    - geowebcache_tiles:/var/local/tiles
+    - geowebcache_tiles:/mnt/geowebcache_tiles
```

Comment:
and : https://github.com/georchestra/georchestra/blob/master/geowebcache-webapp/src/docker/docker-entrypoint.d/00-adjust-perms#L3